src/cdk/scrolling/virtual-scroll-viewport.scss

```diff
@@ -69,11 +73,7 @@ cdk-virtual-scroll-viewport {
 // set if it were rendered all at once. This ensures that the scrollable content region is the
 // correct size.
 .cdk-virtual-scroll-spacer {
-  position: absolute;
```

Comment:
Can you explain more about why this changed? Were these styles just unnecessary or they were somehow interfering?

Reply:
With these styles in place, `Content after virtual scrolling items` was overlapping the `cdk-virtual-scroll-viewport`. Removing them solved the issue.
```
<div class="demo-viewport" cdk-virtual-scrollable-element>
  <p>Content before virtual scrolling items</p>
  <cdk-virtual-scroll-viewport itemSize="50">
    <div *cdkVirtualFor="let size of fixedSizeData; let i = index" class="demo-item"
         [style.height.px]="size">
      Item #{{i}} - ({{size}}px)
    </div>
  </cdk-virtual-scroll-viewport>
  <p>Content after virtual scrolling items</p>
</div>
```